.travis.yml

```diff
@@ -0,0 +1,37 @@
+language: node_js
+
+node_js:
+  - "0.10"
+
+services:
+  - mongodb
+
+cache:
+  directories:
+    - leaderboard-nightwatch/.meteor/local/build/programs/server/assets/packages
+    - ~/.meteor
+#    - node_modules
+#    - webapp/node_modules
+
+before_install:
+  # set up the node_modules dir, so we know where it is
+  - "mkdir -p node_modules &"
+  # install nightwatch, so we know where it is
+  - "npm install nightwatch"
+  # fire up xvfb on port :99.0
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+  # set the xvfb screen size to 1280x1024x16
+  - "/sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1280x1024x16"
+  # install meteor
+  - "curl https://install.meteor.com | /bin/sh"
+  # setup Meteor app
+  - "cd leaderboard-nightwatch"
+    # Optionally we can update all our dependencies here
+    #- "meteor update"
+  - "meteor reset"
+  - "meteor -p 3000 &"
+  # give Meteor some time to download packages, init data, and to start
+  - "sleep 50"
+
+script: ./run_nightwatch.sh
```

Comment:
So this sits in the root of the app and will need to have each framework added to it then, @awatson1978?